src/TitleDetails/TitleDetailsContainer.js

```diff
@@ -113,6 +116,18 @@ const TitleDetailsContainer = ({ location, history, mutator, match, resources })
     [fetchReceivingResources, showCallout, titleId],
   );
 
+  useEffect(() => {
+    setLocations();
+
+    if (pieces && poLine.id) {
```

Comment:
why poLine.id ? what if poLine is undefined?

Reply:
the initial state for poLine is `{}`